src/v-st-ext.adoc

```diff
@@ -274,19 +274,18 @@ register-resident vectors.
 Implementations must provide fractional LMUL settings that allow the
 narrowest supported type to occupy a fraction of a vector register
 corresponding to the ratio of the narrowest supported type's width to
-that of the largest supported type's width.  In general, the
-requirement is to support LMUL {ge} SEW~MIN~/ELEN, where SEW~MIN~ is
-the narrowest supported SEW value and ELEN is the widest supported SEW
-value.  In the standard extensions, SEW~MIN~=8.  For
-standard vector extensions with ELEN=32, fractional LMULs of 1/2 and
-1/4 must be supported.  For standard vector extensions with ELEN=64,
-fractional LMULs of 1/2, 1/4, and 1/8 must be supported.
-
-NOTE: When LMUL < SEW~MIN~/ELEN, there is no guarantee
+that of the vector register width.  In general, the requirement is to 
+support LMUL {ge} SEW~MIN~/VLEN, where SEW~MIN~ is the narrowest supported
+SEW value and VLEN is the length of vector register.  In the standard
+extensions, SEW~MIN~=8.  For standard vector extensions with VLEN=32,
```

Comment:
This changed text expresses different requirements than the original text and, in essence, is a material change to the ratified spec.  For example, it no longer expresses a requirement for implementations with VLEN>64 to support fractional LMULs.

At the same time, this change seem sunnecessary since an implementation with VLEN>64 is already free to support e8mf8.  The requirement expressed in the ELEN=32 example does not imply a limit on mf8 support when at least one full 8-bit element can be supported (i.e. with VLEN>32).  It is just expressing minimum requirements.

And, in fact, the initial statement "Implementations must provide ..." effectively expesses a requirement that e8mf8 must be  supported in VLEN>32 implementations since such implementations can hold at least one 8-bit element in a vector register.  And the second sentence expresses a general minimum requirement - which the further sentences expand on with concrete examples.

Now I would agree that the first sentence could be clarified a little bit.  For example saying "Implementations must provide fractional LMUL settings that allow **at least one element of** the narrowest supported type to occupy a fraction of a vector register corresponding to the ratio of the narrowest supported type's width to that of the largest supported type's width